packages/itwinui-css/src/tabs/tabs.scss

```
@@ -191,3 +204,58 @@
     }
   }
 }
+
+@mixin iui-tabs-actions-wrapper {
+  grid-area: tabs-actions;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  // To always prevent actions from sticking to content
+  --_iui-borderless-horizontal-tabs-actions-content-gap: var(--iui-size-3xs);
+  --_iui-borderless-vertical-tabs-actions-content-gap: var(--iui-size-xs);
+
+  // Add larger min actions-content spacing for borderless horizontal tabs
+  :where(.iui-tabs-wrapper.iui-horizontal) .iui-borderless + & {
```

should not target `iui-borderless` without `iui-tabs`
```suggestion
  :where(.iui-tabs-wrapper.iui-horizontal) :where(.iui-tabs.iui-borderless) + & {
```